app.py

```diff
@@ -452,6 +454,25 @@ def get_first_name(schedule):
     return schedule.get("preferred_name") or schedule["firstname"]
 
 
+def get_github_info():
+    # this uses PyGithub module
+    # using an access token from a person who can access epschedule
+    g = gh(os.environ["GH_TOKEN"])
```

Comment:
The issue here is that the unit test runner doesn't have a GH_TOKEN, and that's probably good because unit tests shouldn't need to actually talk to the GitHub API. For now I think the simplest solution would be to add a line like 
```
if ["GH_TOKEN"] not in os.environ:
    return []
````